packages/icons/src/library/heading.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const heading = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="-2 -2 24 24">
```

Comment:
We're keeping the viewbox hack to keep Dashicons sharp, yet 24x24px in dimensions, correct? To be clear it seems a very solid hack and a good way to transition to bigger more legible icons.

Reply:
yes